src/foam/nanos/fs/File.js

```diff
@@ -202,9 +202,15 @@ foam.CLASS({
       name: 'labels',
       class: 'StringArray',
       documentation: 'List of labels applied to this file',
+      validateObj: function(labels) {
+        if (labels.indexOf("") >= 0) {
+          return 'An assigned file label cannot be empty';
```

Comment:
I originally used message but it just output the message id instead of the message string.
Is there something i need to do to pull the message insteady

I had return 'INVALID_FILE_LABEL' and INVALID_FILE_LABEL was displayed

Reply:
you would need to return `this.INVALID_FILE_LABEL` as a constant

Reply:
fixed with message + translation